package.json

```diff
@@ -155,6 +155,7 @@
     "react-router-dom": "^5.2.0",
     "react-test-renderer": "^17.0.1",
     "remark": "^13.0.0",
+    "resize-observer": "^1.0.2",
```

Comment:
Something is off with this. The page completely froze. Did you consider using the built-in API (giving up on the older version of Safari)?

Reply:
Yes, but I get this error: `ReferenceError: ResizeObserver is not defined`. Do you have any clue?

Reply:
I see that `resize-observer-polyfill` is in `yarn.lock` but not in `package.json`. Maybe some team member tried this package in the past. For consistency and because it is more widely used, I removed `resize-observer` and installed `resize-observer-polyfill` instead!